src/main/java/org/broadinstitute/consent/http/resources/Resource.java

```diff
@@ -51,11 +52,11 @@ protected Response createExceptionResponse(Exception e) {
             if (handler != null) {
                 return handler.handle(e);
             } else {
-                return Response.serverError().entity(new Error(e.getMessage(), Response.Status.INTERNAL_SERVER_ERROR.getStatusCode())).build();
+                return Response.serverError().type(MediaType.APPLICATION_JSON).entity(new Error(e.getMessage(), Response.Status.INTERNAL_SERVER_ERROR.getStatusCode())).build();
```

Comment:
We need to force errors to JSON for the cases when a non-json endpoint fails. In those cases, Jersey tries to force the error into the non-json format and then fails, again.